### PR TITLE
Update exports to belong to user

### LINKED ISSUE #61: Update exports model to belong_to a user and the inverse relation
This is so we can track who is requesting the export.  Update the exports controller to add in the current_user to the attributes

### PR DESCRIPTION
**Pull request by grimm**
Fixes #61

### CODE REVIEW

**Comment by grimm** [comment]
Ready for review

I get an error when I try to download an export, basically says it can't find the file for download, issue with development?  Or?

File: app/models/export.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class Export < ActiveRecord::Base
   serialize :variables, JSON
 
+  belongs_to :users
```
Comment (teknofire) [inline app/models/export.rb]: should be `belongs_to :user`

**Comment by grimm** [comment]
Ready for review

File: app/models/user.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 class User < ActiveRecord::Base
   include GinaAuthentication::UserModel
+
+  has_many :exports
```
Comment (teknofire) [inline app/models/user.rb]: should also have dependent destroy, if user gets deleted it needs to clear their exports as well.

**Comment by grimm** [comment]
Ready for review

**Comment by teknofire** [comment]
👍

**Pull request merged**